custom_components/enphase_envoy_custom/translations/nl.json

```diff
@@ -2,7 +2,7 @@
     "config": {
         "abort": {
             "already_configured": "Apparaat is al geconfigureerd",
-            "reauth_successful": "Herauthenticatie was succesvol"
+            "reauth_successful": "Authenticatie vernieuwing was succesvol"
```

Comment:
I prefer the old version.

Reply:
No strong preference for either here so undo this one seems best then.

Reply:
reverted in next commit